tests/src/test_helpers/ssl_helpers.c

```diff
@@ -46,6 +43,8 @@ void mbedtls_test_init_handshake_options(
     mbedtls_test_handshake_test_options *opts)
 {
 #if defined(MBEDTLS_SSL_HANDSHAKE_WITH_CERT_ENABLED)
+    int rng_seed = 0xBEEF;
+
     srand(rng_seed);
     rng_seed += 0xD0;
```

Comment:
Now that `rng_seed` has moved from file-`static` to being a local variable, this line does nothing. I presume the intent is that different calls to `mbedtls_test_init_handshake_options()` would get different sequences of random numbers, but I don't know why. Whatever the reason, it's not happening now, and this line should either be removed, or `rng_seed` should be made `static` within this function

Reply:
I've restored the current behavior.